Library/Homebrew/cleaner.rb

```diff
@@ -76,6 +76,12 @@ def executable_path?(path)
     path.text_executable? || path.executable?
   end
 
+  # Both these files are completely unnecessary to package and cause
+  # pointless conflicts with other formulae. They are removed by Debian,
+  # Arch & MacPorts amongst other packagers as well. The files are
+  # created as part of installing any Perl module.
+  PERL_BASENAMES = Set.new(%w[perllocal.pod .packlist]).freeze
```

Comment:
What's the benefit to using `Set.new(%w[...])` instead of just `%w[...]`

Reply:
`Set#include?` is faster than `Array#include?`.